webknossos/webknossos/cli/convert_zarr.py

```diff
@@ -37,14 +37,12 @@
 
 
 def _try_open_zarr(path: UPath) -> tensorstore.TensorStore:
-    try:
-        return tensorstore.open(
-            {"driver": "zarr3", "kvstore": {"driver": "file", "path": path}}
-        ).result()
-    except tensorstore.TensorStoreError:
-        return tensorstore.open(
-            {"driver": "zarr", "kvstore": {"driver": "file", "path": path}}
-        ).result()
+    return tensorstore.open(
+        {
+            "driver": "auto",
```

Comment:
I assume this merging of the two codepaths with "auto" is not part of the fix, but instead a refactoring using the auto feature of tensorstore now?